### PR TITLE
QRadar: fix mirroring queue continue to grow

### PR DESCRIPTION
**Pull request by ilaner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-23983
## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/QRadar/ReleaseNotes/2_4_19.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### IBM QRadar v3
+
+- Fixed an issue where the `fetch_incidents` command added to the mirroring queue even though `MIRROR_OFFENSES_WITH_EVENTS` was set to `false`.
```
Comment (tkatzir) [inline Packs/QRadar/ReleaseNotes/2_4_19.md:6]:
```suggestion
- Fixed an issue where the `fetch_incidents` command added offenses to the mirroring queue, even though `MIRROR_OFFENSES_WITH_EVENTS` was set to `false`.
```

File: Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py [hunk 2]
```diff
@@ -2055,6 +2065,7 @@ def long_running_execution_command(client: Client, params: Dict):
     incident_type = params.get('incident_type')
     mirror_options = params.get('mirror_options', DEFAULT_MIRRORING_DIRECTION)
     mirror_direction = MIRROR_DIRECTION.get(mirror_options)
+    mirror_options = params.get('mirror_options', '')
```
Comment (tkatzir) [inline Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py:2068]: Not sure I understand the logic here.

**Review by tkatzir — COMMENTED** [review]
Looks good. Added comment(s)

**Comment by DeanArbel** [comment]
@ShirleyDenkberg Please review the release notes for this PR.

**Comment by ShirleyDenkberg** [comment]
@DeanArbel @tkatzir Doc review completed.

**Comment by ilaner** [comment]
Force merge is needed because of failing QRadar Hunting test which is unrelated

**Pull request merged**